ush/config.aqm.nco.realtime.yaml

```diff
@@ -0,0 +1,104 @@
+metadata:
+  description: config for Online-CMAQ, AQM_NA_13km, real-time, NCO mode on WCOSS2
+user:
+  RUN_ENVIR: nco
+  MACHINE: wcoss2
+  ACCOUNT: [account name]
+workflow:
+  USE_CRON_TO_RELAUNCH: true
+  CRON_RELAUNCH_INTVL_MNTS: 3
+  EXPT_SUBDIR: aqm_nco_aqmna13km
+  PREDEF_GRID_NAME: AQM_NA_13km
+  CCPP_PHYS_SUITE: FV3_GFS_v16
+  DATE_FIRST_CYCL: '2023013000'
+  DATE_LAST_CYCL: '2023013018'
+  INCR_CYCL_FREQ: 6
+  FCST_LEN_HRS: -1
+  FCST_LEN_CYCL:
+    - 6
+    - 72
+    - 72
+    - 6
+  PREEXISTING_DIR_METHOD: rename
+  VERBOSE: true
+  COMPILER: intel
+  DIAG_TABLE_TMPL_FN: diag_table_aqm.FV3_GFS_v16
+  FIELD_TABLE_TMPL_FN: field_table_aqm.FV3_GFS_v16
+  DO_REAL_TIME: true
+#  COLDSTART: false
+#  WARMSTART_CYCLE_DIR: /path/to/restart/dir
+nco:
+  envir: prod
+  NET: aqm
+  model_ver: v7.0
+  RUN: aqm_nco_aqmna13km
+#  OPSROOT: /path/to/custom/opsroot
```

Comment:
Are these commented out lines required? If not, please remove commented out line 35.

Reply:
Same as above. Activated instead